src/facades/wordpress.php

```diff
@@ -12,9 +12,18 @@ function whip_wp_check_versions( $requirements ) {
 			return;
 		}
 
+		global $wp_version;
+
+
 		$config  = include dirname( __FILE__ ) . '/../configs/default.php';
 		$checker = new Whip_RequirementsChecker( $config );
 
+		$dismisser = new Whip_MessageDismisser( $wp_version, new Whip_WPDismissOption() );
+
+		if ( ! $dismisser->isDismissed() ) {
```

Comment:
This does not read logically: "If the message has not been dismissed, stop execution."
It seems that the function is returning the inverted expected value. False should be True.